### DISCUSSION TITLE
Vercel is serving an old version of the site after changing the connected repository.

### DISCUSSION THREAD

**Discussion by ethereal-squirrel** [vercel/community#4656] [Help]
### Summary

**Summary**

Vercel is presenting the old version of my site to a number of browsers (Safari on iOS, Chrome on Windows) if they've ever visited the site before. Clearing browser cache and data cache within Vercel does not solve anything.

**Background**

The new application is built with Next.js 13 using the app router, and is hosted on Vercel.
The previous version of the application was a Vite driven React application.

**Problem**

When visiting the application (https://givin.gifts/) in Chrome, it shows the old version of the site (completely different design and feature set). If I do a hard refresh, it shows the new version. If I then do a normal refresh, it shows the old version again.

When the old version is displayed, X-Vercel-Cache has a HIT, when it displays the new version, this is a MISS.

-----

Any idea what's going on?

### Steps to Reproduce

N/A

**Comment by amyegan** [comment]
Hey @ethereal-squirrel. Are you still having this problem?
